pkg/printers/internalversion/printers.go

```diff
@@ -248,47 +220,210 @@ func AddHandlers(h printers.PrintHandler) {
+		{Name: "Count", Type: "string", Description: apiv1.Event{}.SwaggerDoc()["count"]},
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "Kind", Type: "string", Description: apiv1.Event{}.InvolvedObject.SwaggerDoc()["kind"]},
+		{Name: "Subobject", Type: "string", Description: apiv1.Event{}.InvolvedObject.SwaggerDoc()["fieldPath"]},
+		{Name: "Type", Type: "string", Description: apiv1.Event{}.SwaggerDoc()["type"]},
+		{Name: "Reason", Type: "string", Description: apiv1.Event{}.SwaggerDoc()["reason"]},
+		{Name: "Source", Type: "string", Description: apiv1.Event{}.SwaggerDoc()["source"]},
+		{Name: "Message", Type: "string", Description: apiv1.Event{}.SwaggerDoc()["message"]},
+	}
+	h.TableHandler(eventColumnDefinitions, printEvent)
+	h.TableHandler(eventColumnDefinitions, printEventList)
+
+	namespaceColumnDefinitions := []metav1alpha1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "Status", Type: "string", Description: "The status of the namespace"},
+		{Name: "Age", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["creationTimestamp"]},
+	}
+	h.TableHandler(namespaceColumnDefinitions, printNamespace)
+	h.TableHandler(namespaceColumnDefinitions, printNamespaceList)
+
+	secretColumnDefinitions := []metav1alpha1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "Type", Type: "string", Description: apiv1.Secret{}.SwaggerDoc()["type"]},
+		{Name: "Data", Type: "string", Description: apiv1.Secret{}.SwaggerDoc()["data"]},
+		{Name: "Age", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["creationTimestamp"]},
+	}
+	h.TableHandler(secretColumnDefinitions, printSecret)
+	h.TableHandler(secretColumnDefinitions, printSecretList)
+
+	serviceAccountColumnDefinitions := []metav1alpha1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "Secrets", Type: "string", Description: apiv1.ServiceAccount{}.SwaggerDoc()["secrets"]},
+		{Name: "Age", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["creationTimestamp"]},
+	}
+	h.TableHandler(serviceAccountColumnDefinitions, printServiceAccount)
+	h.TableHandler(serviceAccountColumnDefinitions, printServiceAccountList)
+
+	persistentVolumeColumnDefinitions := []metav1alpha1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "Capacity", Type: "string", Description: apiv1.PersistentVolumeSpec{}.SwaggerDoc()["capacity"]},
+		{Name: "AccessModes", Type: "string", Description: apiv1.PersistentVolumeSpec{}.SwaggerDoc()["accessModes"]},
+		{Name: "ReclaimPolicy", Type: "string", Description: apiv1.PersistentVolumeSpec{}.SwaggerDoc()["persistentVolumeReclaimPolicy"]},
```

Comment:
@smarterclayton Are names meant to be camel case, or e.g. it would be better to have it as `Reclaim Policy` here? In the original printer it was `RECLAIMPOLICY`.